utils/utils.jsx

```diff
@@ -829,7 +829,11 @@ export function changeCss(className, classValue) {
     if (className.indexOf('@media') >= 0) {
         mediaQuery = '}';
     }
-    styleSheet.insertRule(className + '{' + classValue + '}' + mediaQuery, styleSheet.cssRules.length);
+    try {
+        styleSheet.insertRule(className + '{' + classValue + '}' + mediaQuery, styleSheet.cssRules.length);
+    } catch (e) {
+        // do nothing
```

Comment:
0/5 but maybe `console.error()` here?

Reply:
had it and removed it just before submitting the PR, let me add it back